editor.planx.uk/src/@planx/components/List/Public/Fields.tsx

```diff
@@ -16,6 +15,7 @@ import { DESCRIPTION_TEXT, ERROR_MESSAGE } from "../../shared/constants";
 import BasicRadio from "../../shared/Radio/BasicRadio";
 import type { NumberField, QuestionField, TextField } from "../model";
 import { useListContext } from "./Context";
+import { get } from "lodash";
```

Comment:
This change was required as the formik's `getIn()` doesn't correctly handle `fn` fields using dot notation (e.g. `proposal.propertyType`).

Docs: https://formik.org/docs/api/fieldarray#fieldarray-validation-gotchas

Reply:
:bulb: